app/questionnaire/questionnaire_schema.py

```diff
@@ -94,11 +96,41 @@ def __init__(
         self._populate_answer_dependencies()
         self._populate_when_rules_section_dependencies()
         self._populate_calculated_summary_section_dependencies()
+        self._populate_mix_max_for_numeric_answers()
 
     @cached_property
     def answer_dependencies(self) -> ImmutableDict[str, set[AnswerDependent]]:
         return ImmutableDict(self._answer_dependencies_map)
 
+    def _append_to_min_max_map(
+        self, min_max: Literal["minimum", "maximum"], answer_id: str, answers: list
```

Comment:
since you're only ever using `answers[0]` you could pass this into the function instead of the list - minor, not fussed

(Why is it that we only need `answers[0]`? I did try editing the schema to have multiple detail option answers and this still worked but not sure if this would get hit by some edge case somewhere)

Reply:
We have a feature called "variants" which means we can have multiple variants for 1 answer. The variant to use is resolved at run time, it is possible 1 answer id might have 2 answers where only 1 has a min/max or a combination. Therefore, the best option is probably to use the largest range. i.e. if answer 1 has no min/max, then the default min limit is 0, the default max is system limit, but if answer 2 has min/max set to -1000 and 1000, we should be using the min from answer 2 (as it is longer) and max from answer 1. Let me know if it needs further explanation.

Reply:
I will need some more logic for that then and for that I think I will still need to pass the list in